ci/pipelines/pull-request/jinja.template.yml

```diff
@@ -264,7 +264,10 @@ jobs:
           GRADLE_TASK_OPTIONS: ""
           MAINTENANCE_VERSION: {{repository.branch}}
           PARALLEL_DUNIT: {{test.PARALLEL_DUNIT}}
+        {% if test.PARALLEL_GRADLE is defined %}
+          # TODO: DO NOT COMMIT THIS CHANGE WITHOUT USING A DEFAULTING TEMPLATE.  Should this default true or false?
```

Comment:
As before, I want parallel to be the default and tested via the pipeline, as the source of truth. Make this happen, and remove the TODO